patches/@segment/analytics-react-native-firebase+1.1.1-beta.3.patch

```diff
@@ -3,24 +3,24 @@ patch-package
 +++ b/node_modules/@segment/analytics-react-native-firebase/android/build.gradle
```

Comment:
Was it intentional to include this change in this PR?

Reply:
This is super old so when I pulled master I was battling the patches-- I think it's gone away now that I am using the correct `yarn.lock`, so this will likely be unchanged from what's on master. Still cleaning things up